search-app/src/cmr/search/site/routes.clj

```diff
@@ -2,19 +2,51 @@
   "This namespace is the one responsible for the routes intended for human
   consumption."
   (:require
-   ;; Third-party libs
-   [compojure.core :refer :all]
-   [ring.swagger.ui :as ring-swagger-ui]
+    ;; Third-party libs
+    [compojure.core :refer :all]
+    [ring.util.response :refer [redirect]]
+    [ring.swagger.ui :as ring-swagger-ui]
 
-   ;; CMR libs
-   [cmr.collection-renderer.api.routes :as collection-renderer-routes]
-   [cmr.common.api.context :as context]
-   [cmr.common-app.api-docs :as api-docs]))
+    ;; CMR libs
+    [cmr.collection-renderer.api.routes :as collection-renderer-routes]
+    [cmr.common.api.context :as context]
+    [cmr.common-app.api-docs :as api-docs]
+    [cmr.search.site.pages :as pages]))
+
+(defn- redir-url
```

Comment:
This is really just a hack. Do you guys have a utility that does something like this?

Or, heck, a better suggestion? :-)